src/main/java/life/qbic/domain/sample/SampleEventDatasource.java

```diff
@@ -10,4 +11,6 @@
 public interface SampleEventDatasource {
   <T extends SampleEvent> void store(T sampleEvent);
   List<SampleEvent> findAllForSample(SampleCode sampleCode);
```

Comment:
NIT: Same as before i think all is quite ambigous and would recommend something like:
findSamplesMatchingSampleCode 
findSamplesForSampleCode

or if that is too long

findMatchingSamples

Reply:
As this is a `SampleEventDatasource`, it handles SampleEvents. I oriented myself on the spring implementations of the datasource. Thus I will not change that.